packages/mu-transport/index.js

```diff
@@ -39,17 +40,47 @@ module.exports = function transport (createDriver, mu, opts) {
     tearDown: driver.tearDown
   }
 
+
+
+  function constructFailResponse (err, msg) {
+    var pkt = { err: err,
+                response: {},
+                protocol: {}}
+    var elt = muid
+
+    /* istanbul ignore else  */
```

Comment:
Please remove that sentence, so we do not forget about it. It's ok if coverages drops consequently.